aws-customerprofiles-calculatedattributedefinition/README.md

```diff
@@ -0,0 +1,12 @@
+# AWS::CustomerProfiles::CalculatedAttributeDefinition
+
+Congratulations on starting development! Next steps:
+
+1. Write the JSON schema describing your resource, `aws-customerprofiles-calculatedattributedefinition.json`
+1. Implement your resource handlers.
+
+The RPDK will automatically generate the correct resource model from the schema whenever the project is built via Maven. You can also do this manually with the following command: `cfn generate`.
+
+> Please don't modify files under `target/generated-sources/rpdk`, as they will be automatically overwritten.
+
+The code uses [Lombok](https://projectlombok.org/), and [you may have to install IDE integrations](https://projectlombok.org/setup/overview) to enable auto-complete for Lombok-annotated classes.
```

Comment:
Are these needed? Or a better question, should we modify this to be more public friendly? Like provide some basic guidance on how to use and links to related resources?

Reply:
Addressed, made this file public friendly.